pages/data-design/branches/pulling-pushing-items-between-branches.mdx

```diff
@@ -0,0 +1,113 @@
+import { Callout } from 'nextra/components';
+
+# Pulling and Pushing Items Between Branches
+
+You can add an item from one branch to another by pulling or pushing them. When that is done, it's the same item on both branches
+where the ID of the item remains the same. 
+- [**Pushing an item**](#pushing-items-to-another-branch-alpha): Adding an item from a branch that a user is currently on, to another branch. 
+- [**Pulling an item**](#pulling-items-from-another-branch-coming-in-march-2025): Adding an item to a branch that the user is currently on, from another branch.
+
+<Callout type="info">
+    Pushing is currently out in alpha and pulling will be available in March 2025. [Contact us](mailto:beta@avo.app)
+ to get access.
+</Callout>
+
+## Pushing items to another branch (alpha)
+
+Pushing events, properties and property bundles from the branch that a user is currently on, to another branch is possible from the Tracking plan changes view. 
+To access the tracking plan changes view, first you have to be on a branch and then you can click "View Changes" under the branch name.
+
+![Navigating to tracking plan changes](/images/pulling-pushing-items/navigating-to-tracking-plan-changes.png)
+
+There is an button in the top right corner of every tracking plan change that can be pushed to another branch.
+
+![Pressing push button](/images/pulling-pushing-items/press-push-button.png)
+
+When that button is pushed, you will get a modal that prompts you to pick a branch to push the changes to and review your selection.
+You have to first pick a branch so that Avo can accurately display how the changes will be applied, as it might differ per branch if the item already exists on the destination branch.
+
+<center><img
+  src="/docs/images/pulling-pushing-items/empty-push-modal.png"
+  width="60%"
+  alt="Empty review modal"
+/></center>
+
+When the destination branch is selected, the actual diff for the destination branch is shown. The items shown can be expanded to show the details of what will be added to the destination branch.
+If the item you selected to push is an event or a property bundle, any related property changes are displayed below. See the [property changes related to event](#property-changes-related-to-event) section for details.
```

Comment:
I think I would put emphasis here on that Avo smartly decides _only_ to take the needed changes, i.e. the smallest set possible.